### PR TITLE
#1019: Update Dependencies on top of 8.3.0 

### LINKED ISSUE #1019: Update Dependencies on top of 8.3.0
Security issues found by the nightly CI-build require updating the following packages incl. recommendations.

## Ubuntu 22.04

```
Package|Requested Version|Candidate Version
curl|7.81.0-1ubuntu1.18|7.81.0-1ubuntu1.19
libcurl4-openssl-dev|7.81.0-1ubuntu1.18|7.81.0-1ubuntu1.19
python3.10-dev|3.10.12-1~22.04.6|3.10.12-1~22.04.7
```

### PR DESCRIPTION
**Pull request by tomuben**
closes #1019 

### All Submissions:

* [x] Check if your pull request goes to the correct bash branch (develop or master)?
* [x] Is the title of the Pull Request correct?
* [x] Is the title of the corresponding issue correct?
* [x] Have you updated the changelog?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../../../pulls) for the same update/change?  
* [x] Are you mentioning the issue which this PullRequest fixes ("Fixes...")

### When integrating to Develop branch:

1. [x] Remember to merge with "Squash and Merge"

### When integrating to Main branch:

1. [ ] Remember to merge with "Merge"
2. [ ] Have you thought about version number? If there is a breaking change in the toolchain, need to update major version.
3. [ ] If you plan a release, have you checked the Exasol packages for updates?
   1. Websocket API (https://github.com/EXASOL/websocket-api)
   2. [Exasol-BucketFS](https://pypi.org/project/exasol-bucketfs/)

### CODE REVIEW

File: doc/changes/changes_9.0.0.md [hunk 1]
```diff
@@ -18,7 +18,8 @@ This release uses version 1.0.0 of the container tool.
 ## Security Issues
 
  - #956: Updated dependencies
- - #1006: Updated Dependencies on top of 8.4.0 
+ - #1006: Updated Dependencies on top of 8.4.0
```
Comment (tkilias) [inline doc/changes/changes_9.0.0.md:21]: 8.4.0 looks weird

Comment (tomuben) [reply doc/changes/changes_9.0.0.md:21]: Yeah, that was actually a mistake

Comment (tomuben) [reply doc/changes/changes_9.0.0.md:21]: Should have been 8.3.0 as well

Comment (tkilias) [reply doc/changes/changes_9.0.0.md:21]:
```suggestion
 - #1006: Updated Dependencies on top of 8.3.0
```

Comment (tomuben) [reply doc/changes/changes_9.0.0.md]: Done

**Review by tkilias — APPROVED** [review]

**Pull request merged**